tests/README.md

```diff
@@ -78,6 +78,41 @@ Use ./local-integration-test.sh
   ./tests/robot.sh tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot
   ```
 
+## Run Docker command tests via makefile target
+
+There exist a makefile target for developers to run the docker command tests locally (not in CI environment) against a pre-deployed VCH.  This is a fast way for contributors to test their potential PR submissions against the CI tests locally before pushing the commit.  There is also another benefit gained from using the makefile target.  With the way it is setup, logs from the run are written out to vic/ci-results, even if the tests fails.  The method described above to run the tests locally with drone has the weakness that a failure in the test can sometimes result in no written logs to help debug the failure.
+
+There are a few requirements before using this makefile target.
+
+1. A VCH must be pre-deployed before calling this makefile target
+2. The makefile target relies on a script that looks for a few more secrets variable, TEST_URL_ARRAY, TEST_USERNAME, TEST_PASSWORD, and TEST_DATASTORE
```

Comment:
The makefile target relies on a script that looks for a few more secrets variables